package.json

```diff
@@ -279,5 +283,11 @@
     "blocking": false,
     "logInfo": false,
     "logError": true
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --cache --fix",
+      "prettier-standard"
```

Comment:
I've used [`pretty-quick`](https://www.npmjs.com/package/pretty-quick) for this before, but just out of habit without actually researching if it is quicker. However since we are using prettier-standard here perhaps that won't work.

Reply:
I tried using pretty-quick without any further configuration and it seemed to completely ignore the rules specified by standard (even saving files in VSCode started format using prettier's default configurations)